File: CPP/PROGRAM/StorageDevice.cpp
#pragma once
#include <iostream>
#include <string>

using namespace std;

class StorageDevice{
    //atribut
    private:
        int kapasitasGB;
        float bacaTulisMBps;

    public:
        //konstruktor
        StorageDevice(){
            this->kapasitasGB = 0;
            this->bacaTulisMBps = 0;
        }

        StorageDevice(int kapasitasGB, float bacaTulisMBps){
            this->kapasitasGB = kapasitasGB;
            this->bacaTulisMBps = bacaTulisMBps;
        }
        
        //getter dan setter
        void setKapasitasGB(int kapasitasGB){
            this->kapasitasGB = kapasitasGB;
        }

        void setBacaTulisMBps(float bacaTulisMBps){
            this->bacaTulisMBps = bacaTulisMBps;
        }

        int getKapasitasGB(){
            return kapasitasGB;
        }

        float getBacaTulisMBps(){
            return bacaTulisMBps;
        }

        ~StorageDevice(){

        }
};

File: PYTHON/PROGRAM/StorageDevice.py
class StorageDevice:
    #konstruktor
    def __init__(self, kapasitasGB = 0, bacaTulisMBps = 0):
        self.__kapasitasGB = kapasitasGB
        self.__bacaTulisMBps = bacaTulisMBps

    #getter dan setter
    def getKapasitasGB(self):
        return self.__kapasitasGB
    
    def setKapasitasGB(self, kapasitasGB):
        self.__kapasitasGB = kapasitasGB

    def getBacaTulisMBps(self):
        return self.__bacaTulisMBps
    
    def setBacaTulisMBps(self, bacaTulisMBps):
        self.__bacaTulisMBps = bacaTulisMBps